__tests__/caesar_cipher.test.js

```diff
@@ -0,0 +1,22 @@
+'use strick'
+import { wrap } from '../src/caesar_cipher'
+import { cipher } from '../src/caesar_cipher'
+
+
+
+describe('Ceaser Cipher Tests', () => {
+  
+  test('word transposition same case', () => {
+    expect(cipher('a', 1)).toBe('b')
+  })
+
+  test('wrap z to a',() => {
+    expect(cipher('z',1)).toBe('a')
+  })
+
+  test('punctuation!',() => {
```

Comment:
This should be `caesarCipher('microverse.>?', 7)).toBe('tpjyvclyzl.>?')`. Don't confuse with the number . I used `7`. The main thing is in result it only works on **a to z or A to Z**. Any punctuation should be unchanged.